### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by diana-shyrokikh**

### CODE REVIEW

File: .emv.sample
Comment (LiudmylaKulinchenko) [inline .emv.sample:1]: remove the file from a github, it is sensitive information

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+    command: >
+      sh -c "python3 manage.py migrate &&
+             python3 manage.py wait_for_db &&
+             python3 manage.py runserver 0.0.0.0:8000"
```
Comment (YuriiKindrat) [inline docker-compose.yml:14]: it seems to me, u should wait for db and then only migrate )

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: .emv.sample
Comment (Oleg-s-s) [reply .emv.sample:1]: Wdym? Its sample for .env variables, its not consisting any sensitive info, just a hint for those ones who will be setting project up on their machines.

File: user/management/commands/wait_for_db.py
Comment (Oleg-s-s) [inline user/management/commands/wait_for_db.py:1]: 1 wait_for_db command will be enough. optimize it

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]
You should keep the .env.example in your repository, and never include your .env file populated with sensitive information such as database credentials and API keys.

The reason being, the .env.example file acts as a “blueprint” for your .env file so any developer who checks the repository out (no matter what branch) should be able to create an .env specific to their environment to be able to work with your project.

long story short, you can return your .env.sample into pr))
